lib/wasix/src/runners/wasi.rs

```diff
@@ -242,6 +242,7 @@ impl WasiRunner {
         let container_fs = if let Some(pkg) = pkg {
             builder.add_webc(pkg.clone());
             builder.set_module_hash(pkg.hash());
+            builder.include_package(pkg.id.clone());
```

Comment:
Isn't this too limited?

A BinaryPackage can consist of multiple dependencies providing file systems, do all of those need to be included here?

(I think this would require an extension to BinaryPackage, because the file system entries don't record their source package IDs, IIRC).

Reply:
Do we have a test that gets FS volumes from multiple dependencies?

Reply:
Yes, makes sense.